doc/docs/Python_Tutorials/Basics.md

```diff
@@ -209,7 +209,7 @@ vals = []
 def get_slice(sim):
     center = mp.Vector3(0, -3.5)
     size = mp.Vector3(16, 0)
-    vals.append(sim.get_array(center=center, size=size, component=mp.Ez))
+    vals.append(sim.get_array(mp.Volume(center, size=size), component=mp.Ez))
```

Comment:
Using `center=x, size=y` in the tutorial seems better, since it is shorter than constructing a `Volume` object.